src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,174 @@
+(ns status-im2.common.standard-auth.view
+  (:require [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.view :as numbered-keyboard]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [react-native.fast-image :as fast-image]
```

Comment:
hmm, I believe we don't want to use fast-image for images like this where it won't change etc and it's coming from our own resources. 
right @ilmotta ? 🤔

Reply:
@J-Son89 Removed `fast-image` usage and used `rn/image` instead of it.

Reply:
Exactly @J-Son89